src/main/webapp/choosebranch.js

```diff
@@ -0,0 +1,17 @@
+/* Purpose: script that fetches recommended career path */
+
+function getRecommendedCareerPath() {
+    const responsePromise = fetch('/careerquiz');
+    // when server request complete, pass response into handleResponse
+    responsePromise.then(handleResponse);
+}
+
+function handleResponse(response) {
+    // receives Java Object (ArrayList reponse)
+    const recommendedCareerPath = response.json();
+    recommendedCareerPath.then(changeDom);
+}
+
+function changeDom(recommendedCareerPath) {
+  console.log(recommendedCareerPath);
+}
```

Comment:
New line. Does the linter catch this?

Reply:
honestly, need to investigate if the linter actually works at all..

Reply:
Changed.